pyvenv.el

```diff
@@ -97,6 +97,16 @@ educated guess, but that can be off."
   :safe #'file-directory-p
   :group 'pyvenv)
 
+(defcustom pyvenv-exec-shell
+  (or (executable-find "sh")
+      (executable-find "bash"))
+  "The path to a POSIX compliant shell to use for running
+  virtualenv hooks. Useful if you use a non-POSIX shell (e.g.
+  fish)."
+  :type '(file :must-match t)
+  :safe #'file-directory-p
```

Comment:
I don't think this is true. Do you think a file you are editing should be able to get pyvenv to execute any and all executable it wants? This variable should not be marked safe at all.

Reply:
AH. Very good point. Thank you, will update.